libtelemetry/src/androidTest/java/com/mapbox/android/telemetry/TelemetryServiceTest.java

```diff
@@ -135,6 +139,17 @@ public void checksOnTaskRemovedCallbackWhenOnTaskRemovedCalled() throws Exceptio
     verify(mockedCallback, times(1)).onTaskRemoved();
   }
 
+  @Test
+  public void checksLocationPermission() throws Exception {
+    Intent serviceIntent = new Intent(InstrumentationRegistry.getTargetContext(), TelemetryService.class);
```

Comment:
what are we trying to test here for? since permission is already granted with `GrantPermissionRule` how useful is it? Should we consider implementing a test when permissions are not present and try to verify the outcome/result with public APIs exposed by `TelemetryService`?